### PR TITLE
Remove deprecated props of ScreenGridLayer from examples and tests.

### PR DESCRIPTION
**Pull request by kaushikb11**
For #2892 

- Remove deprecated props `minColor` and `maxColor` of ScreenGridLayer from examples.

- Modify test cases for ScreenGridLayer.

### CODE REVIEW

**Comment by kaushikb11** [comment]
Modified `screengrid-infoviz.png` image for a test case of ScreenGridLayer.

Screenshot from 2019-04-13 02-55-01

File: test/render/test-cases.js [hunk 1]
```diff
@@ -779,28 +777,6 @@ export const TEST_CASES = [
     ],
     goldenImage: './test/render/golden-images/grid-lnglat.png'
   },
-  {
```
Comment (Pessimistress) [inline test/render/test-cases.js]: Why is this one deleted? We need tests for both GPU aggregation and CPU aggregation.

Comment (kaushikb11) [reply test/render/test-cases.js]: GPU aggregation is covered in the `screengrid-lnglat-colorRange` test case as default aggregation happens on GPU.

Comment (kaushikb11) [reply test/render/test-cases.js]: There was an extra test case as it had `minColor` and `maxColor` props.

**Pull request merged**